### PR TITLE
Fix `verify-format-sql` CI check not reporting SQL formatting issues

### PR DESCRIPTION
**Pull request by sean-rose**
It turns out that ever since Jinja support was added to `bqetl` in March (#3691) the `verify-format-sql` CI check effectively hasn't been verifying anything, because it's checking the _generated_ SQL, which has `bqetl query render` run on everything, which Jinja-renders and _formats_ all queries.

This fixes the `verify-format-sql` CI check to verify the format of the original SQL, and also fixes the SQL formatting issues that have accumulated so the fixed `verify-format-sql` CI check will pass.

---
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2393)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_last_seen_v1/query.sql
Comment (sean-rose) [inline sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_last_seen_v1/query.sql:42]: I suspect some of these empty comments were added specifically to have the preceding block formatted as multi-line rather than inlined.  If there are cases where forcing multi-line block formatting is desired the current solution would be to add a line comment within the block.

File: sql/moz-fx-data-shared-prod/subscription_platform_derived/active_subscriptions_v1/query.sql
Comment (sean-rose) [inline sql/moz-fx-data-shared-prod/subscription_platform_derived/active_subscriptions_v1/query.sql:2]: This isn't strictly necessary, but it served as a good test of the fix to Jinja comment formatting, and I think the only reason the comment wasn't put here in the first place is `bqetl format` was misformatting it to combine it with the preivous line, like
```sql
WITH product_union AS ({#- format off #}
```

Comment (sean-rose) [inline sql/moz-fx-data-shared-prod/subscription_platform_derived/active_subscriptions_v1/query.sql:2]: This isn't strictly necessary, but it served as a good test of the fix to Jinja comment formatting, and I think the only reason the comment wasn't put here in the first place is `bqetl format` was misformatting it to combine it with the preivous line, like
```sql
WITH product_union AS ({#- format off #}
```

File: sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_revenue_levers_daily_v1/query.sql
Comment (sean-rose) [inline sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_revenue_levers_daily_v1/query.sql:1]: @alekhyamoz would you mind reviewing this as a representative of the [revenue_forecasting_data_reviewers](https://github.com/orgs/mozilla/teams/revenue_forecasting_data_reviewers) team?  This is the file modification that triggered a required approval from that team per [CODEOWNERS](https://github.com/mozilla/bigquery-etl/blob/c3fa65a30efbf18b37b5bacaa622f6ff29a78b86/CODEOWNERS#L18), and it's just formatting changes, no functional changes.

Comment (skahmann3) [reply sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_revenue_levers_daily_v1/query.sql:1]: @alekhyamoz is on PTO through February 1. We also work closely with @curtismorales, though he would need to be formally added to the revenue forecasting data reviewers group

**Review by scholtzan — APPROVED** [review]

File: sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_revenue_levers_daily_v1/query.sql
Comment (sean-rose) [reply sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_revenue_levers_daily_v1/query.sql:1]: @skahmann3 you could also approve this PR.  I was just pinging Alekhya specifically since she's a fellow data engineer and I've worked with her before.

**Review by skahmann3 — APPROVED** [review]

**Pull request merged**
